oar/core/advisory_mgr.py

```diff
@@ -145,7 +145,7 @@ def push_to_cdn_staging(self):
             ads = self.get_advisories()
             for ad in ads:
                 if not ad.are_all_push_jobs_completed():
-                    ad.push_to_cdn()
+                    ad.push_to_cdn("")
```

Comment:
@wewang58 
```suggestion
                    ad.push_to_cdn("stage")
```
although default target is `stage`, use explicit value it is more meaningful